fedmsg_meta_fedora_infrastructure/bz.py

```diff
@@ -145,6 +150,12 @@ def _gather_emails(self, msg):
 
         return users
 
+    def packages(self, msg, **config):
+        component = msg['msg']['bug']['component']
+        if component in bz_components_that_are_not_packages:
+            return set()
```

Comment:
hotness has had a lot of trouble trying to do that :(

We could try.. but lots of people use different/non-standard formats when they type in the info.